### ISSUE TITLE
Problem with query contents

### ISSUE THREAD

**Issue by raeray** [sqlfluff/sqlfluff#3416]
### Search before asking

- [X] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

Hello! When i run `sqlfluff fix` i have the following error:

```To hide this warning, add the failing file to .sqlfluffignore
Traceback (most recent call last):
  File "/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/linter/runner.py", line 103, in run
    yield partial()
  File "/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/linter/linter.py", line 693, in lint_rendered
    return cls.lint_parsed(
  File "/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/linter/linter.py", line 619, in lint_parsed
    tree, initial_linting_errors, ignore_buff = cls.lint_fix_parsed(
  File "/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/linter/linter.py", line 545, in lint_fix_parsed
    new_tree, _ = tree.apply_fixes(
  File "/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/parser/segments/base.py", line 1161, in apply_fixes
    s, fixes = seg.apply_fixes(dialect, rule_code, fixes)
  File "/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/parser/segments/base.py", line 1161, in apply_fixes
    s, fixes = seg.apply_fixes(dialect, rule_code, fixes)
  File "/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/parser/segments/base.py", line 1161, in apply_fixes
    s, fixes = seg.apply_fixes(dialect, rule_code, fixes)
  [Previous line repeated 6 more times]
  File "/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/parser/segments/base.py", line 1165, in apply_fixes
    r = r.__class__(
  File "/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/parser/segments/base.py", line 206, in __init__
    pos_marker = PositionMarker.from_child_markers(
  File "/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/parser/markers.py", line 105, in from_child_markers
    raise ValueError("Attempted to make a parent marker from multiple files.")
ValueError: Attempted to make a parent marker from multiple files.```

Not sure exactly what is going on. I"m a bit hesitant to 

### Expected Behaviour

Fix the syntax of my query

### Observed Behaviour

The error stack trace says to post the error here

### How to reproduce

This is the query i was running:

```
with invoices_by_customer1 as (
select
    ws_customer_id
    , max(customer_name) as customer_name
    , date_trunc('month', invoice_created_at) as month
    , max(invoice_created_at) as max_transaction
    , sum(net_amount_total) as net_amount_saas
from
    {{ ref('financial_transactions') }}
where
	net_amount_total
	> 0
	and invoice_created_at <= dateadd('hour', 12, convert_timezone('America/Los_Angeles', current_date()))
group by
	ws_customer_id, date_trunc(
	'month', invoice_created_at)
)

select *
	, row_number() over (partition by ws_customer_id order by month asc) as asc_row
	, row_number() over (partition by ws_customer_id order by month desc) as desc_row
	, lead(month) over (partition by ws_customer_id order by month asc) as next_transaction_month
	, lag(month) over (partition by ws_customer_id order by month asc) as prev_transaction_month
	, case
		  when next_transaction_month = dateadd('month', 1, month)
			  then lead(net_amount_saas) over (partition by ws_customer_id order by month asc)
		  else 0
	end as next_month_saas
	, case
		  when prev_transaction_month = dateadd('month', -1, month)
			  then lag(net_amount_saas) over (partition by ws_customer_id order by month asc)
		  else 0
	end as prev_month_saas
	, case
		  when next_transaction_month != dateadd('month', 1, month)
				  and dateadd('hour', 12, dateadd('month', 1, max_transaction)) <
				  convert_timezone('America/Los_Angeles', current_date())
			  then true
		  else false
	end as churned
	, case when asc_row = 1 then true else false end as new_client
	, case
		  when next_transaction_month is null and prev_transaction_month is null
				  and dateadd('month', 1, max_transaction) < convert_timezone('America/Los_Angeles', current_date())
			  then true
		  else false
	end as paid_once
	, case
		  when prev_transaction_month is not null and prev_transaction_month != dateadd('month', -1, month)
			  then true
		  else false
	end as revived
	, case
		  when next_month_saas > net_amount_saas
			  then true
		  else false
	end as upsell
	, case
		  when next_month_saas < net_amount_saas and next_month_saas is not null and next_month_saas != 0
			  then true
		  else false
	end as downsell
	, case
		  when net_amount_saas > prev_month_saas
				  and month = dateadd('month', 1, prev_transaction_month)
				  and prev_month_saas in ('1', '5', '10')
				  and asc_row > 2
			  then true
		  else false
	end as out_of_data_retention
	, case
		  when prev_month_saas > net_amount_saas
				  and dateadd('month', 1, prev_transaction_month) <
				  convert_timezone('America/Los_Angeles', current_date())
				  and net_amount_saas in ('1', '5', '10')
				  and asc_row > 2
			  then true
		  else false
	end as into_data_retention
	, case
		  when paid_once = true and churned = true
			  then 'paid_once_churn'
		  when churned = true
			  then 'churn'
		  when revived = true
			  then 'revived'
		  when new_client = true
			  then 'new_client'
		  when into_data_retention = true
			  then 'into_data_retention_downsell'
		  when downsell = true
			  then 'downsell'
		  when out_of_data_retention = true
			  then 'out_of_data_retention_upsell'
		  when upsell = true
			  then 'upsell'
	end as label
    , current_timestamp last_updated

from
	invoices_by_customer1
```

### Dialect

snowflake

### Version

0.11.1

### Configuration

 [sqlfluff:rules]
 templater = dbt
 dialect = snowflake
 tab_space_size = 4
 max_line_length = 120
 comma_style = leading
 allow_scalar = False
 single_table_references = consistent
 capitalisation_policy = lower

 [sqlfluff:rules:L011]
 # Aliasing preference for tables
 aliasing = implicit

 [sqlfluff:rules:L012]
 # Aliasing preference for columns
 aliasing = implicit

 [sqlfluff]
 exclude_rules = L027, L031, L011, L012


### Are you willing to work on and submit a PR to address the issue?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)

**Comment by mdahlman** [comment]
Hi @raeray ,
This issue is not a bug but rather some sort of configuration issue. Instead of logging it, you should ask for help on the [SQLFluff Slack channel](https://join.slack.com/t/sqlfluff/shared_invite/zt-o1f4x0e8-pZzarAIlQmKj_6ZwD16w0g).

FYI: I ran your query through sqlfluff, and it parsed and fixed just fine.
```
==== fixing violations ====
136 fixable linting violations found
Are you sure you wish to attempt to fix these? [Y/n]
```
I attached the result.
[issue-3416-fixed.txt](https://github.com/sqlfluff/sqlfluff/files/8825907/issue-3416-fixed.txt)

This issue should get closed as "Won't Fix" or "Not a bug" or something like that.

**Comment by tunetheweb** [comment]
Personally I'd prefer people opened a GitHub issue, especially when providing a complex query and config like here. Slack is more for quick questions rather than specific issues like this one.

Saying that, it also worked fine for me. I don't have dbt installed, but since there was no dbt-specific code in the query, I switched `templater = dbt` to `templater = jinja` and then attempted to `sqlfluff fix` the file and it fixed fine for me:

```
% sqlfluff fix test.sql -f --fixed-suffix FIXED
==== finding fixable violations ====
== [test.sql] FAIL                                                                                                                                                                 
L:   2 | P:   1 | L003 | Indent expected and not found compared to line #1
L:   8 | P:   1 | L003 | Indent expected and not found compared to line #1
L:  10 | P:   1 | L003 | Indent expected and not found compared to line #1
L:  11 | P:   1 | L004 | Incorrect indentation type found in file.
L:  12 | P:   1 | L004 | Incorrect indentation type found in file.
L:  13 | P:   1 | L004 | Incorrect indentation type found in file.
L:  14 | P:   1 | L003 | Indent expected and not found compared to line #1
L:  15 | P:   1 | L004 | Incorrect indentation type found in file.
L:  16 | P:   1 | L004 | Incorrect indentation type found in file.
L:  16 | P:   2 | L003 | Indent expected and not found compared to line #15
L:  19 | P:   1 | L036 | Select targets should be on a new line unless there is
                       | only one select target.
L:  20 | P:   1 | L004 | Incorrect indentation type found in file.
L:  21 | P:   1 | L004 | Incorrect indentation type found in file.
L:  22 | P:   1 | L004 | Incorrect indentation type found in file.
L:  23 | P:   1 | L004 | Incorrect indentation type found in file.
L:  24 | P:   1 | L004 | Incorrect indentation type found in file.
L:  25 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  25 | P:   1 | L004 | Incorrect indentation type found in file.
L:  25 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  26 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  26 | P:   1 | L004 | Incorrect indentation type found in file.
L:  26 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  27 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  27 | P:   1 | L004 | Incorrect indentation type found in file.
L:  27 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  28 | P:   1 | L004 | Incorrect indentation type found in file.
L:  29 | P:   1 | L004 | Incorrect indentation type found in file.
L:  30 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  30 | P:   1 | L004 | Incorrect indentation type found in file.
L:  30 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  31 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  31 | P:   1 | L004 | Incorrect indentation type found in file.
L:  31 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  32 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  32 | P:   1 | L004 | Incorrect indentation type found in file.
L:  32 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  33 | P:   1 | L004 | Incorrect indentation type found in file.
L:  34 | P:   1 | L004 | Incorrect indentation type found in file.
L:  35 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  35 | P:   1 | L004 | Incorrect indentation type found in file.
L:  35 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  35 | P:  10 | L043 | Unnecessary CASE statement. Use COALESCE function
                       | instead.
L:  36 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  36 | P:   1 | L004 | Incorrect indentation type found in file.
L:  36 | P:   7 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  37 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  37 | P:   1 | L004 | Incorrect indentation type found in file.
L:  37 | P:   7 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  38 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  38 | P:   1 | L004 | Incorrect indentation type found in file.
L:  38 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  39 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  39 | P:   1 | L004 | Incorrect indentation type found in file.
L:  39 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  40 | P:   1 | L004 | Incorrect indentation type found in file.
L:  41 | P:   1 | L004 | Incorrect indentation type found in file.
L:  41 | P:  14 | L043 | Unnecessary CASE statement. Use COALESCE function
                       | instead.
L:  42 | P:   1 | L004 | Incorrect indentation type found in file.
L:  43 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  43 | P:   1 | L004 | Incorrect indentation type found in file.
L:  43 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  43 | P:  10 | L043 | Unnecessary CASE statement. Use COALESCE function
                       | instead.
L:  44 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  44 | P:   1 | L004 | Incorrect indentation type found in file.
L:  44 | P:   7 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  45 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  45 | P:   1 | L004 | Incorrect indentation type found in file.
L:  45 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  46 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  46 | P:   1 | L004 | Incorrect indentation type found in file.
L:  46 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  47 | P:   1 | L004 | Incorrect indentation type found in file.
L:  48 | P:   1 | L004 | Incorrect indentation type found in file.
L:  49 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  49 | P:   1 | L004 | Incorrect indentation type found in file.
L:  49 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  49 | P:  10 | L043 | Unnecessary CASE statement. Use COALESCE function
                       | instead.
L:  50 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  50 | P:   1 | L004 | Incorrect indentation type found in file.
L:  50 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  51 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  51 | P:   1 | L004 | Incorrect indentation type found in file.
L:  51 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  52 | P:   1 | L004 | Incorrect indentation type found in file.
L:  53 | P:   1 | L004 | Incorrect indentation type found in file.
L:  54 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  54 | P:   1 | L004 | Incorrect indentation type found in file.
L:  54 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  54 | P:  10 | L043 | Unnecessary CASE statement. Use COALESCE function
                       | instead.
L:  55 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  55 | P:   1 | L004 | Incorrect indentation type found in file.
L:  55 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  56 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  56 | P:   1 | L004 | Incorrect indentation type found in file.
L:  56 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  57 | P:   1 | L004 | Incorrect indentation type found in file.
L:  58 | P:   1 | L004 | Incorrect indentation type found in file.
L:  59 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  59 | P:   1 | L004 | Incorrect indentation type found in file.
L:  59 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  59 | P:  10 | L043 | Unnecessary CASE statement. Use COALESCE function
                       | instead.
L:  60 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  60 | P:   1 | L004 | Incorrect indentation type found in file.
L:  60 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  61 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  61 | P:   1 | L004 | Incorrect indentation type found in file.
L:  61 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  62 | P:   1 | L004 | Incorrect indentation type found in file.
L:  63 | P:   1 | L004 | Incorrect indentation type found in file.
L:  64 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  64 | P:   1 | L004 | Incorrect indentation type found in file.
L:  64 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  64 | P:  10 | L043 | Unnecessary CASE statement. Use COALESCE function
                       | instead.
L:  65 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  65 | P:   1 | L004 | Incorrect indentation type found in file.
L:  65 | P:   7 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  66 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  66 | P:   1 | L004 | Incorrect indentation type found in file.
L:  66 | P:   7 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  67 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  67 | P:   1 | L004 | Incorrect indentation type found in file.
L:  67 | P:   7 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  68 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  68 | P:   1 | L004 | Incorrect indentation type found in file.
L:  68 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  69 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  69 | P:   1 | L004 | Incorrect indentation type found in file.
L:  69 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  70 | P:   1 | L004 | Incorrect indentation type found in file.
L:  71 | P:   1 | L004 | Incorrect indentation type found in file.
L:  72 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  72 | P:   1 | L004 | Incorrect indentation type found in file.
L:  72 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  72 | P:  10 | L043 | Unnecessary CASE statement. Use COALESCE function
                       | instead.
L:  73 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  73 | P:   1 | L004 | Incorrect indentation type found in file.
L:  73 | P:   7 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  74 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  74 | P:   1 | L004 | Incorrect indentation type found in file.
L:  74 | P:   7 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  75 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  75 | P:   1 | L004 | Incorrect indentation type found in file.
L:  75 | P:   7 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  76 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  76 | P:   1 | L004 | Incorrect indentation type found in file.
L:  76 | P:   7 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  77 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  77 | P:   1 | L004 | Incorrect indentation type found in file.
L:  77 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  78 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  78 | P:   1 | L004 | Incorrect indentation type found in file.
L:  78 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  79 | P:   1 | L004 | Incorrect indentation type found in file.
L:  80 | P:   1 | L004 | Incorrect indentation type found in file.
L:  81 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  81 | P:   1 | L004 | Incorrect indentation type found in file.
L:  81 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  82 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  82 | P:   1 | L004 | Incorrect indentation type found in file.
L:  82 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  83 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  83 | P:   1 | L004 | Incorrect indentation type found in file.
L:  83 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  84 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  84 | P:   1 | L004 | Incorrect indentation type found in file.
L:  84 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  85 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  85 | P:   1 | L004 | Incorrect indentation type found in file.
L:  85 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  86 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  86 | P:   1 | L004 | Incorrect indentation type found in file.
L:  86 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  87 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  87 | P:   1 | L004 | Incorrect indentation type found in file.
L:  87 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  88 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  88 | P:   1 | L004 | Incorrect indentation type found in file.
L:  88 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  89 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  89 | P:   1 | L004 | Incorrect indentation type found in file.
L:  89 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  90 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  90 | P:   1 | L004 | Incorrect indentation type found in file.
L:  90 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  91 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  91 | P:   1 | L004 | Incorrect indentation type found in file.
L:  91 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  92 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  92 | P:   1 | L004 | Incorrect indentation type found in file.
L:  92 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  93 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  93 | P:   1 | L004 | Incorrect indentation type found in file.
L:  93 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  94 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  94 | P:   1 | L004 | Incorrect indentation type found in file.
L:  94 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  95 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  95 | P:   1 | L004 | Incorrect indentation type found in file.
L:  95 | P:   5 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  96 | P:   1 | L002 | Mixed Tabs and Spaces in single whitespace.
L:  96 | P:   1 | L004 | Incorrect indentation type found in file.
L:  96 | P:   6 | L003 | Indentation not hanging or a multiple of 4 spaces
L:  97 | P:   1 | L004 | Incorrect indentation type found in file.
L: 101 | P:   1 | L004 | Incorrect indentation type found in file.
==== fixing violations ====
204 fixable linting violations found
FORCE MODE: Attempting fixes...
Persisting Changes...
== [test.sql] PASS
Done. Please check your files to confirm.
  [3 unfixable linting violations found]
```

@raeray can you repeat that and confirm it works for you? And explain more about how to repeat the error you got?

**Comment by raeray** [comment]
Hm, I still get the same error, but i was able to find a workaround.
i put a print where the error was in the file `/Users/<user_name>/.pyenv/versions/3.9.1/envs/venv_workstream/lib/python3.9/site-packages/sqlfluff/core/parser/markers.py` and i found that it was breaking when trying to apply the fix for Rule [L043](https://docs.sqlfluff.com/en/stable/rules.html#sqlfluff.core.rules.Rule_L043) so i manually refactored my case statements into coalesce and the code runs now.

i think the code was not able to apply the fix, maybe because there are many and statements. However, i don't understand how it works for you guys and not for me...
